resolwe_bio/tools/basespace_download.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+"""Tool to download files from BaseSpace."""
+
+import sys
+import atexit
+import argparse
+import requests
+
+
+class BaseSpaceDownloadError(Exception):
```

Comment:
Populate this class to report errors.

Reply:
What do you mean? This subclass is used to distinguish errors handled in the download tool. The errors are being reported by instantiating the error class with a string in the constructor, e.g. https://github.com/genialis/resolwe-bio/pull/466/files#diff-5d9c604384eec6edf05f1ec1f946a57cR54

Reply:
You're correct, I missed something.